src/test/resources/job_response.json

```diff
@@ -0,0 +1,27 @@
+{
+    "info": [
+        {
+            "StartTime": "2015, Mar 04 19:28:17.363806",
+            "Target": "*",
+            "Target-type": "glob",
+            "User": "johnny",
+            "Arguments": [],
+            "Function": "test.ping",
+
+            "jid": "20150304192817363806",
+            "Minions": [
+                "blackbox"
+            ],
+            "Result": {
+                "blackbox": {
+                    "return": true
+                }
+            }
+        }
+    ],
+    "return": [
+        {
+            "blackbox": true
+        }
+    ]
+}
```

Comment:
missing newline here i'm not sure if checkstyle catches this yet for ressources (maybe something to add in #53)